x-pack/legacy/plugins/lens/public/xy_visualization_plugin/types.ts

```diff
@@ -61,15 +59,6 @@ const axisConfig: { [key in keyof AxisConfig]: ArgumentType<AxisConfig[key]> } =
     types: ['string'],
     help: 'The axis title',
   },
-  showGridlines: {
```

Comment:
It makes sense that we don't want to support them in the first release, but we could leave them in on the expression level and just pass in defaults for now. This might be a pattern - we don't have to expose every single thing in the UI, but it should be possible on the expression.

Just a suggestion, I'm also happy with removing it for now and revisiting this later.

Reply:
I see your point, but I'm not yet confident in either direction. The most flexible API is probably to wrap the elastic-charts library in an expression DSL, that way other teams can use this too- that feels like the better long-term plan to me.

Reply:
I'm a fan of removing unused stuff. If we need it in the future, we can always add it back.